src/packages/web/page.gleam

```diff
@@ -76,6 +87,17 @@ fn package_list_item(package: PackageSummary) -> Node(t) {
       ),
       html.h2([], [external_link_text(url, package.name)]),
       html.p_text([], package.description),
+      case links {
+        [] -> html.span([], [])
+        links ->
+          html.div(
+            [attrs.class("package-links")],
+            links
+            |> list.map(fn(link) {
+              html.span([attrs.style("margin-right: var(--gap);")], [link])
```

Comment:
No inline styles please

Reply:
You must have caught this _right_ before I pushed a change :smile:

Inline styles removed in 978aa014a92f0fc17711bcd4ff1b9527b23ee553.